Editor/Modules/TextureModule.cs

```diff
@@ -0,0 +1,89 @@
+        TextureCompression,
+        MipMapEnabled,
+        Readable,
+        Resolution,
+        SizeOnDisk,
+        Num
+    }
+
+    class TextureModule : ProjectAuditorModule
+    {
+        public static string[] searchTheseFolders;
+        public TextureImporter TextProp;
+
+        private static readonly IssueLayout k_IssueLayout = new IssueLayout
+        {
+            category = IssueCategory.Texture,
+            properties = new[]
+            {
+                new PropertyDefinition {type = PropertyTypeUtil.FromCustom(TextureProperties.Name), format = PropertyFormat.String, name = "Name", longName = "Texture Name" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Shape), format = PropertyFormat.String, name = "TextureShape", longName = "Texture Shape" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.ImporterType), format = PropertyFormat.String, name = "Importer Type", longName = "Texture Importer Type" },
+                new PropertyDefinition {type = PropertyTypeUtil.FromCustom(TextureProperties.Format), format = PropertyFormat.String, name = "Format", longName = "Texture Format" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.TextureCompression), format = PropertyFormat.String, name = "Compression Used?", longName = "Texture Compression" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.MipMapEnabled), format = PropertyFormat.Bool, name = "MipMaps", longName = "Texture MipMaps Enabled" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Readable), format = PropertyFormat.Bool, name = "Readable", longName = "Readable" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Resolution), format = PropertyFormat.String, name = "Resolution", longName = "Texture Resolution" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.SizeOnDisk), format = PropertyFormat.String, name = "Size", longName = "Texture Size" },
+            }
+        };
+        public override bool IsEnabledByDefault() { return false; }
+
+        public override IEnumerable<IssueLayout> GetLayouts() {  yield return k_IssueLayout;  }
+
+        public override void Audit(ProjectAuditorParams projectAuditorParams, IProgress progress = null)
+        {
+            var allTextures = AssetDatabase.FindAssets("t: Texture, a:assets");
+            var issues = new List<ProjectIssue>();
+            progress?.Start("Finding Textures", "Search in Progress...", allTextures.Length);
+
+            foreach (var aTexture in allTextures)
+            {
+                var pathToTexture = AssetDatabase.GUIDToAssetPath(aTexture);
+                var tName = ((Texture2D)AssetDatabase.LoadAssetAtPath(pathToTexture, typeof(Texture2D)));
```

Comment:
1. delete extra "(" and ")" 
2. 
![image](https://user-images.githubusercontent.com/108890606/188007502-5158994b-a81e-463e-827b-f499d372c118.png)
For example, this line will have error when aTexture is RenderTexture. Result: tName is null.

Reply:
Corrected the extra parenthesis, and #2 will be fixed at a later time.

Reply:
Fixed #2, so that Texture3D, TextureCube, Texture2D, & Texture2DArray all are processed properly and appear corrects in the Analysis results. 
TextureRenderer has also been ignored as discussed